certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -91,11 +91,11 @@
               }
           }
       },
-      "/api/v3/ip_addresses/{ip}": {
+      "/api/v3/ip_addresses/connectorV2/{ip}": {
```

Comment:
Instead of modifying the the current version, create a new operation version, add it to the family and deprecate the old version (this one) so it doesn't show up for new connections.

Here is a the documentation on how to do it: https://docs.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation